src/wallet/rpcwallet.cpp

```diff
@@ -2274,8 +2274,7 @@ UniValue z_listunspent(const UniValue& params, bool fHelp)
             "    \"type\" : \"sprout|sapling|orchard\", (string) The shielded pool\n"
```

Comment:
Should this be `\"transparent|sprout|sapling|orchard\", (string) the pool of this unspent output`?
Also, if this RPC can return transparent outputs, the description should allow for that rather than being specific to shielded notes.